docs/scripts/prtvol2csv.rst

```diff
@@ -34,13 +34,15 @@ The script will read numbers from the line with ``CURRENTLY IN PLACE`` in Eclips
   ====================================================================================================================================
 
 
-Additionally, if the Eclipse DATA file includes::
+Additionally, if the Eclipse DATA file includes
+
+.. code-block:: none
 
   RPTSOL
     FIP=2 'FIPRESV' /
 
-the PRT-file will also contain a table with pore volumes pr. phase and pr.
-FIPNUM, which will be added to the exported table.
+the PRT-file will also contain a table with pore volumes pr. phase and pr. FIPNUM, 
+which will be added to the exported table. FIPRESV will include reservoir volumes (RM3).
```

Comment:
Minor rewrite suggestion:

> the PRT file will also contain a table with pore volume per phase and per FIPNUM, 
which will be added to the exported table. FIPRESV will include reservoir volumes (RM3).

Reply:
Agree, done.